onnxruntime/core/optimizer/qdq_transformer/qdq_util.cc

```diff
@@ -58,5 +58,31 @@ bool IsQDQPairSupported(
          *q_scale.data<float>() == *dq_scale.data<float>();
 }
 
+bool IsDQSupported(
```

Comment:
>IsDQSupported [](http://example.com/codeflow?start=5&length=13)

code duplicates with the above. Is it better to change this function to 'IsQDQSupported' because Q and DQ logic are same, and then call IsQDQSupported for Q and DQ nodes in the above function.

Reply:
Because of the "check Q/DQ have same scale and zero point" logic in IsQDQPairSupported, we need to keep q_input_defs/dq_input_defs/keepq_zp_tensor_proto/q_scale_tensor_proto/dq_zp_tensor_proto/dq_scale_tensor_proto in the function. We would need q_input_defs/dq_input_defs/keepq_zp_tensor_proto/q_scale_tensor_proto/dq_zp_tensor_proto/dq_scale_tensor_proto initialization in IsQDQSupported caller, which is another duplication of code.